service/title_formatter.py

```diff
@@ -31,6 +35,20 @@ def format_display_json(title_json):
     return title
 
 
+def _format_a1_notes(title_number):
+    # grabs official copy data, extracts the a register sub register and then takes the notes
+    official_copy_data = api_client.get_official_copy_data(title_number)
+    sub_registers = official_copy_data.get('official_copy_data', {}).get('sub_registers')
+    notes = []
+    # Only iterates through if there are sub registers
+    if sub_registers is not None:
+        a_register = next((item for item in sub_registers if item["sub_register_name"] == "A"))
```

Comment:
next() only appears to be called once here, which makes it rather redundant I believe.